monai/transforms/spatial/dictionary.py

```diff
@@ -2174,17 +2174,23 @@ def __init__(
         self,
         keys: KeysCollection,
         grid: Tuple[int, int] = (2, 2),
-        size: Optional[Union[int, Tuple[int, int]]] = None,
+        size: Optional[Union[int, Tuple[int, int], Dict[Hashable, Union[int, Tuple[int, int], None]]]] = None,
```

Comment:
May I know why you change the `size` to be a Dict?
I didn't see the use case to define different size for keys.
And the corresponding doc-string of this arg says it's a tuple.

Thanks.

Reply:
You can see the use case in the tutorial: https://github.com/Project-MONAI/tutorials/blob/27717a16a143a035f889c75e8fb6e7bad3cbffcc/pathology/tumor_detection/ignite/camelyon_train_evaluate.py#L84

I'll update the docstring.